core/src/test/java/greencity/service/impl/TipsAndTricksServiceImplTest.java

```diff
@@ -111,4 +147,18 @@ void delete() {
         tipsAndTricksService.delete(1L);
         verify(tipsAndTricksRepo, times(1)).deleteById(1L);
     }
+
+    @Test
+    void search() {
+        List<TipsAndTricks> tipsAndTricks = Collections.singletonList(ModelUtils.getTipsAndTricks());
+        PageRequest pageRequest = PageRequest.of(0, 3);
+        Page<TipsAndTricks> page = new PageImpl<>(tipsAndTricks, pageRequest, tipsAndTricks.size());
+        List<SearchTipsAndTricksDto> dtoList = page.stream()
+            .map(t -> modelMapper.map(t, SearchTipsAndTricksDto.class))
+            .collect(Collectors.toList());
+        PageableDto<SearchTipsAndTricksDto> pageableDto = new PageableDto<>(dtoList, dtoList.size(), 0);
+        when(tipsAndTricksRepo.searchTipsAndTricks(pageRequest, tipsAndTricks.get(0).getTitle())).thenReturn(page);
+        when(modelMapper.map(tipsAndTricks.get(0), SearchTipsAndTricksDto.class)).thenReturn(dtoList.get(0));
+        assertEquals(pageableDto, tipsAndTricksService.search(tipsAndTricks.get(0).getTitle()));
```

Comment:
The same in each test. The statement

`tipsAndTricksService.search(tipsAndTricks.get(0).getTitle())`

should be on separate line.